DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html

```diff
@@ -0,0 +1,122 @@
+        <label for="legalName" class="dol-form-question-text" id="legalNameLabel">Has this employer ever previously held a 14(c) certificate?</label>
+        <span class="usa-input-error-message" role="alert" ng-show="validate('employer.hasTradeName')">{{ validate('employer.hasTradeName') }}</span>
+        <ul class="usa-unstyled-list">
+          <li>
+            <input id="hasTradeName_1" type="radio" name="hasTradeName" aria-describedby="hasTradeNameHelp" ng-value="true" ng-model="formData.employer.hasTradeName">
+            <label for="hasTradeName_1">Yes</label>
+          </li>
+          <li>
+            <input id="hasTradeName_0" type="radio" name="hasTradeName" ng-value="false" ng-model="formData.employer.hasTradeName">
+            <label for="hasTradeName_0">No</label>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div ng-show="formData.employer.hasTradeName" ng-class="vm.einError || vm.einRequired || vm.invalidEin ? 'usa-input-error' : ''">
+      <label for="certificateNumber">What is the most recently held certificate number for the main establishment?</label>
+      <input class="ein sidelabeled" name="EIN" id="EIN" type="text" mask="99-99999-9-999" ng-model="formData.employer.certificateNumber" />
+      <span class="example">Example: 12-34567-H-890</span>
+    </div>
+    <div class="dol-form-question-block" ng-class="validate('employer.legalName') ? 'usa-input-error' : ''">
+      <div class="dol-form-question-help">
+        <label for="legalName" class="dol-form-question-text" id="legalNameLabel">Legal Name of Employer</label>
+      </div>
+      <span class="usa-input-error-message" role="alert" ng-show="validate('employer.legalName')">{{ validate('employer.legalName') }}</span>
+      <input id="legalName" name="legalName" aria-describedby="legalNameHelp" type="text" ng-model="formData.employer.legalName">
+     <span><a role="button" ng-keyup="$event.keyCode === 13 ? toggleAllHelpText() : null" ng-click="toggleAllHelpText()" tabindex="0">What if the Employer's legal name changed since its last application?</a></span><helplink aria-controls="legalNameHelp" aria-describedby="legalNameLabel"></helplink>
+      <helptext id="legalNameHelp">
+        <p>Use the Employer's legal name as it appeared on the last application.</p>
+        <p>You will have an opportunity to update the Employer's legal name when completing a new application.</p>
+      </helptext>
+    </div>
+
+    <div ng-class="vm.einError || vm.einRequired || vm.invalidEin ? 'usa-input-error' : ''">
+      <label for="EIN">Employer Identification Number (EIN)</label>
+      <span class="usa-input-error-message" id="input-error-message" role="alert" ng-show="vm.einError || vm.einRequired || vm.invalidEin">
+        <div ng-show="vm.einError">
+          <p>The EIN you provided is already registerd. Please contact the account admin for access.</p>
+          <p>If you do not know the account admin, contact our office at 000-123-4567</p>
```

Comment:
Need to get the real contact information here, I guess?

Reply:
@EStriegel what should I use here?